src/components/site/SummaryList.js

```diff
@@ -1,20 +1,29 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 const SummaryList = props => {
   const { items = [] } = props;
   return (
     <div className="dg_summary-list">
       {items.map(item => (
-        <div key={item.key} className="dg_summary-list__item">
-          <i className={item.icon}></i>
-          <div>
-            <h3>{item.heading} </h3>
-            <div dangerouslySetInnerHTML={{ __html: item.content }} />
+        <div key={item.id} className="dg_summary-list__item">
+          <i key={`summaryListItemIcon${item.id}`} className={item.icon}></i>
```

Comment:
sorry, key can also be removed, form the children, just needs to be on the parent div.